nvflare/apis/impl/task_manager.py

```diff
@@ -12,13 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import logging
 from enum import Enum
 from typing import Tuple
 
 from nvflare.apis.controller_spec import ClientTask, Task, TaskCompletionStatus
 from nvflare.apis.fl_context import FLContext
 from nvflare.apis.shareable import Shareable
+from nvflare.fuel.utils.obj_utils import get_logger
```

Comment:
It was originally defined in obj_utils since it was only for objects. I can move it to log_utils since we'll also be adding get_module_logger, get_cls_logger etc